src/components/metadata.js

```diff
@@ -23,7 +23,7 @@ function Metadata(image, gltf, usdz, name, target) {
   script.type = "application/ld+json"
   script.text = data.replace(/^\s*[\r\n]/gm, '') // Remove empty lines caused by missing params
 
-  target.appendChild(script)
+  target?.appendChild(script)
```

Comment:
I guess we could return early in this file as well? Before creating script element in memory, etc.